File: lambda/java/src/main/java/org/dunno/Handler.java
package org.dunno;

import com.amazonaws.services.lambda.runtime.Context;
import com.amazonaws.services.lambda.runtime.RequestHandler;

@SuppressWarnings("unused")
public class Handler implements RequestHandler<Void, String> {
  @Override
  public String handleRequest(Void unused, Context context) {
    System.out.printf(
        "AwsRequestId: %s, FunctionArn: %s",
        context.getAwsRequestId(), context.getInvokedFunctionArn());
    return "OK";
  }
}

File: lambda/quarkus/app/src/main/kotlin/org/dunno/Handler.kt
package org.dunno

import com.amazonaws.services.lambda.runtime.Context
import com.amazonaws.services.lambda.runtime.RequestHandler

class Handler : RequestHandler<Void, String> {
  override fun handleRequest(unused: Void?, context: Context?): String {
    return context?.logStreamName ?: "unknown"
  }
}
